### PR TITLE
Indicate admins on dev login

### PR DESCRIPTION
**Pull request by werebus**
This would have been nice when working on #171

### CODE REVIEW

File: spec/models/roster_spec.rb [hunk 1]
```diff
@@ -147,4 +147,21 @@
...
+
+  describe 'user_options' do
+    let(:roster) { create :roster }
+    let!(:admins) { [roster_admin(roster)] }
+    let!(:non_admins) { [roster_user(roster)] }
+    let(:call) { roster.user_options }
+
+    it 'has admins in the "Admins"' do
+      expect(call.fetch('Admins'))
+        .to contain_exactly(*admins.map { |a| [a.full_name, a.id] })
```
Comment (jforgue) [inline spec/models/roster_spec.rb:159]: Wouldn't `expect(call.fetch('Admins')).to eq(admins.map { |a| [a.full_name, a.id] })` be sufficient?

**Review by jforgue — APPROVED** [review]
Looks good.

**Review by Anbranin — APPROVED** [review]

File: spec/models/roster_spec.rb (hunk 1 above)
Comment (jforgue) [reply spec/models/roster_spec.rb:159]: or maybe `match_array` would be the more appropriate matcher?

Comment (Anbranin) [reply spec/models/roster_spec.rb:159]: What's wrong with `contain_exactly`? I use it all the time so I'm curious

Comment (werebus) [reply spec/models/roster_spec.rb:159]: There are two different improvement paths @jforgue proposed here. In the first, he was pointing out that, in this test, there is only one admin, and a one-element array can only be in one order, so `eq` would work here. However, I prefer to explicitly say "in no particular order", I think.

In the second, he suggests `match_array` which would get rid of the need for the splat.  That is, these two expectations are the same:

```ruby
expect(a_thing).to contain_exactly(:a, :b, :c)
expect(a_thing).to match_array([:a, :b, :c])
```

**Pull request merged**